include/random_walks/nuts_hmc_walk.hpp

```diff
@@ -0,0 +1,384 @@
+// VolEsti (volume computation and sampling library)
```

Comment:
Have you also tried to involve the average number of reflections on the burn-in method?

Reply:
No, I did not. I added a TODO comment to generalize Nesterov's algorithm in the truncated setting.